### ISSUE TITLE
About melcloud child bridge

### ISSUE THREAD

**Issue by peibol81** [grzegorz914/homebridge-melcloud-control#118]
I have the plugin configured with the child bridge option in homebridge and it works well for me. But when I enter the home app, I go to settings and then to the section of centrals of accessories and bridges the nelcloud bridge appears to me, but if I enter it does not appear that the unifad mitsubishi is connected to it. It's a little strange because it works well and I don't know if it's normal. Screenshot attachment![image](https://github.com/grzegorz914/homebridge-melcloud-control/assets/159157043/44bc4625-e1c0-439d-80d7-4bad20ab0c30)

**Comment by grzegorz914** [comment]
https://github.com/grzegorz914/homebridge-melcloud-control/wiki#adding-to-homekit

**Comment by peibol81** [comment]
Sorry. Is not necesary the melcloud bridge?El 26 may 2024, a las 15:44, Grzegorz ***@***.***> escribió:﻿
https://github.com/grzegorz914/homebridge-melcloud-control/wiki#adding-to-homekit

**Comment by grzegorz914** [comment]
You need to add this accessory itself, the bridge is irrelevant by external dynamic platform accessory.

**Comment by peibol81** [comment]
Thank you.

If I have understood correctly, can I delete the bridge in the home app? And on the other hand, in the Homebridge configuration I have configured the plugin with the child bridge option, is this correct? I think the latter is, but please confirm me because I don't understand. Finally, in my case, is it necessary to use the option to unpair devices within the Homebridge UI options? I'm sending a screenshot so you can understand what I mean. Climate control is the Mitsubishi unit, homebridge- melcloud the child bridge and the other is the main homebridg bridge, which I do not use, should I remove any devices from that list? Thank you for your time and apologies for my clumsiness![image](https://github.com/grzegorz914/homebridge-melcloud-control/assets/159157043/94173325-1089-4231-87cf-d3d4361933da)

**Comment by grzegorz914** [comment]
For more info about child bridge you can read this https://github.com/homebridge/homebridge/wiki/Child-Bridges
In your case You don’t need delete the bridge from home app but You can if You want. In homebridge itself the bridge is automatically create if you run plugin as a child bridge and do not remove it from there, Child bridge prevent crash the whole homebridge system if something happens in plugin itself. In home app you need to add the accessory itself (Climatizador D6D6) to display it in home.

**Comment by peibol81** [comment]
Thank you very much. I've already understood it. The configuration of this plugin is a little different from what I had understood, but it is already clear. I have finally deleted the melcloud bridge that appeared in the home app because I like to have everything organized and configured in the most optimal way possible. I'm a bit of a maniac with these things... Thank you very much for everything. You have solved my doubts that I have been dragging since I installed the plugin.
